io.github.archisman_panigrahi.QuickBib.json

```diff
@@ -0,0 +1,94 @@
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "url": "https://files.pythonhosted.org/packages/92/8d/e296c7af03757debd8fc80df2898cbed4fb69fc61ed2c9b4a1d42e923a9e/bibtexparser-1.4.3.tar.gz",
+          "sha256": "a9c7ded64bc137720e4df0b1b7f12734edc1361185f1c9097048ff7c35af2b8f"
+        },
+        {
+          "type": "file",
+          "url": "https://files.pythonhosted.org/packages/e4/37/af0d2ef3967ac0d6113837b44a4f0bfe1328c2b9763bd5b1744520e5cfed/certifi-2025.10.5-py3-none-any.whl",
+          "sha256": "0f212c2744a9bb6de0c56639a6f68afe01ecd92d91f14ae897c4fe7bbeeef0de"
+        },
+        {
+          "type": "file",
+          "url": "https://files.pythonhosted.org/packages/13/69/33ddede1939fdd074bce5434295f38fae7136463422fe4fd3e0e89b98062/charset_normalizer-3.4.4.tar.gz",
+          "sha256": "94537985111c35f28720e43603b8e7b43a6ecfb2ce1d3058bbe955b73404e21a"
+        },
+        {
+          "type": "file",
+          "url": "https://files.pythonhosted.org/packages/b8/6f/65aa57114505d96d316c49a117bdf540b4ecb5fb2eeae0165465bd119a29/doi2bib3-0.3.1-py3-none-any.whl",
+          "sha256": "ad06728806d59466e9f4e1fe48cce5e4ec603f910bb19a16c038db67d4494530"
+        },
+        {
+          "type": "file",
+          "url": "https://files.pythonhosted.org/packages/0e/61/66938bbb5fc52dbdf84594873d5b51fb1f7c7794e9c0f5bd885f30bc507b/idna-3.11-py3-none-any.whl",
+          "sha256": "771a87f49d9defaf64091e6e6fe9c18d4833f140bd19464795bc32d966ca37ea"
+        },
+        {
+          "type": "file",
+          "url": "https://files.pythonhosted.org/packages/10/5e/1aa9a93198c6b64513c9d7752de7422c06402de6600a8767da1524f9570b/pyparsing-3.2.5-py3-none-any.whl",
+          "sha256": "e38a4f02064cf41fe6593d328d0512495ad1f3d8a91c4f73fc401b3079a59a5e"
+        },
+        {
+          "type": "file",
+          "url": "https://files.pythonhosted.org/packages/1e/db/4254e3eabe8020b458f1a747140d32277ec7a271daf1d235b70dc0b4e6e3/requests-2.32.5-py3-none-any.whl",
+          "sha256": "2462f94637a34fd532264295e186976db0f5d453d1cdd31473c85a6a161affb6"
+        },
+        {
+          "type": "file",
+          "url": "https://files.pythonhosted.org/packages/a7/c2/fe1e52489ae3122415c51f387e221dd0773709bad6c6cdaa599e8a2c5185/urllib3-2.5.0-py3-none-any.whl",
+          "sha256": "e6b01673c0fa6a13e374b50871808eb3bf7046c4b125b216f6bf1cc604cff0dc"
+        }
+      ]
+    },
+    {
+      "name": "quickbib",
+      "buildsystem": "simple",
+      "build-commands": [
+        "meson setup builddir --prefix=/app",
+        "meson compile -C builddir",
+        "meson install -C builddir"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/archisman-panigrahi/QuickBib.git",
+          "commit": "a55bcb87f3a81213772fcde6e0c80595d4f8f166"
+        }
+      ]
+    }
+  ],
+  "finish-args": [
```

Comment:
finish-args should be in the top you can place it before `command:`

Reply:
Fixed in https://github.com/flathub/flathub/pull/7143/commits/171e3ff41aaabb8110fd72219b9a4915a3cde05b